dom_control/lib/dom_control.rb

```diff
@@ -0,0 +1,9 @@
+module DomControl
```

Comment:
would love to actually change the namespace to`DOMControl`, but ran into autoloading issues with previous attempts getting this working, but I think the `Zeitwerk::Registry` approach should now make it naming agnostic... probably...